client/app/telecom/telephony/alias/configuration/feature/contactCenterSolution/sounds/telecom-telephony-alias-configuration-sounds.html

```diff
@@ -196,7 +192,7 @@ <h2 data-translate="telephony_alias_config_contactCenterSolution_sounds_title"><
         </oui-button>
         <oui-button data-variant="primary"
                     data-on-click="$ctrl.updateContactCenterSolution()"
-                    data-disabled="!$ctrl.hasChanged() || $ctrl.soundsHoldForm.$invalid || $ctrl.soundsOverflowForm.$invalid || $ctrl.soundsClosureForm.$invalid || !$ctrl.queueOptions.actionOnClosureParam || !$ctrl.queueOptions.actionOnOverflowParam">
+                    data-disabled="!$ctrl.hasChanged() || $ctrl.soundsHoldForm.$invalid || $ctrl.soundsOverflowForm.$invalid || $ctrl.soundsClosureForm.$invalid">
```

Comment:
Just so you know, the guidelines say that the user should be able to click on the button even if the form is invalid but the validation is triggered on click

Reply:
Ok good to know